docs/meshstack.aws.index.md

```diff
@@ -30,380 +29,219 @@ When configuring these roles, operators must take care to correctly guard agains
-        "RoleName": "AWSCloudFormationStackSetAdministrationRole",
-        "AssumeRolePolicyDocument": {
-          "Version": "2012-10-17",
-          "Statement": [
-            {
-              "Effect": "Allow",
-              "Principal": {
-                "Service": "cloudformation.amazonaws.com"
-              },
-              "Action": "sts:AssumeRole"
-            }
-          ]
-        },
-        "Path": "/",
-        "Policies": [
-          {
-            "PolicyName": "AssumeRole-AWSCloudFormationStackSetExecutionRole",
-            "PolicyDocument": {
-              "Version": "2012-10-17",
-              "Statement": [
-                {
-                  "Sid": "VisualEditor0",
-                  "Effect": "Allow",
-                  "Action": "sts:AssumeRole",
-                  "Resource": "arn:aws:iam::*:role/AWSCloudFormationStackSetExecutionRole"
-                }
-              ]
-            }
-          }
-        ]
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::<<MESHCLOUD_ACCOUNT_ID>>:user/meshfed-service-user"
+      },
+      "Action": "sts:AssumeRole",
+      "Condition": {
+        "StringEquals": {
+          "sts:ExternalId": "<<EXTERNAL_ID>>"
+        }
       }
     }
-  }
+  ]
 }
 ```
 
-### Account Access
+Replace `MESHCLOUD_ACCOUNT_ID` with the dedicated meshcloud AWS account id where the `meshfed-service-user` lives. Replace `EXTERNAL_ID` accordingly as well.
 
-The following operations are performed via meshStack in the deployed AWS accounts:
+### Automation Account Setup
 
-* Verifying SAML IDP integrity/update
-* Account Alias Setup
+The automation account should contain a `MeshfedAutomationRole`.
 
-The default access to AWS accounts is done via a assumed role to `MeshstackAccountAccessRole`. This role should have the following minimal policy attached:
+The following policy and trust relationship should be attached to the role so that `meshfed-service-user` can assume it in order to roll out CloudFormation stack instances in the newly provisioned accounts or to invoke a Lambda that would trigger account bootstrapping.
 
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Policy-->
 ```json
 {
-  "ManagedPolicyName": "MeshcloudMinimalServicePolicy",
-  "Description": "Minimal Access for meshstack Automation Service",
-  "Roles": [
-    "MeshstackAccountAccessRole"
-  ],
-  "PolicyDocument": {
     "Version": "2012-10-17",
     "Statement": [
-      {
-        "Effect": "Allow",
-        "Action": [
-          "iam:CreateSAMLProvider",
-          "iam:GetSAMLProvider",
-          "iam:UpdateSAMLProvider",
-          "iam:DeleteSAMLProvider",
-          "iam:ListSAMLProviders"
-        ],
-        "Resource": [
-          "arn:aws:iam::<ACCOUNT_ID>:saml-provider/*",
-          "arn:aws:cloudformation:*:<ACCOUNT_ID>:stack/meshstack-cf-access*"
-        ]
+        {
+            "Sid": "VisualEditor0",
```

Comment:
d: This sid is looking a bit akward. Can we change it to something more meaningful? It's also set at other places. If you replace it, please replace it everywhere

Reply:
Will change.